src/styles/index.scss

```diff
@@ -55,4 +55,307 @@ body {
     margin: 0 auto;
     position: relative;
   }
+
```

Comment:
Not a fan of specifying these globally. Give the component some static classname and wrap these into its own file under the slider folder. Similar to `CategoryNavigation.scss`.

Reply:
I have tried to use slick as a partial, but there is a problem in where scss modules dont apply nested global styles, this is something that can be done is a separate PR later